driver/Clflags.ml

```diff
@@ -33,6 +33,7 @@ let option_falignfunctions = ref (None: int option)
 let option_falignbranchtargets = ref 0
 let option_faligncondbranchs = ref 0
 let option_finline_asm = ref false
+let option_mcpu = ref "generic"
```

Comment:
Following on my previous proposal, option_mcpu would be initialized to Configuration.model.